### ISSUE TITLE
Decoding?

### ISSUE THREAD

**Issue by gijoegotyler** [aemkei/aurebesh.js#5]
Is there anyway to turn it back into regular JS?

**Comment by programmeruser2** [comment]
Yes. First evaluate the assignments at the beginning which provide the values needed to evaluate the string. Then evaluate the value in the parentheses right before the final `()`.
